Tools/simulation/gz/models/advanced_plane/model.sdf

```diff
@@ -286,7 +286,7 @@
         <geometry>
           <mesh>
             <scale>0.1 0.1 0.1</scale>
-            <uri>model://rc_cessna/meshes/rudder.dae</uri>
+            <uri>https://fuel.gazebosim.org/1.0/frede791/models/Advanced%20Plane/tip/files/meshes/rudder.dae</uri>
```

Comment:
Good for now, but would like to put this under the PX4 org in ignition fuel

Reply:
Yes I talked with @dagar about this. He added me as a member to the fuel PX4 team on Friday so I will make a PR to adjust all of these.